### PR TITLE
Update release.json and spec.json for all blocks

### PR DESCRIPTION
**Pull request by hansmosh**

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -8,5 +8,10 @@
     "Language": "Python",
     "Reference": "github.com/nio-blocks/snmp.git@v1.0.0",
     "Version": "1.0.0"
+  },
+  "nio/SNMPWalk": {
```
Comment (dorzel) [inline release.json]: This is a duplicate entry

File: spec.json [hunk 2]
```diff
@@ -89,5 +89,8 @@
         },
         "Category": "Communication",
         "Commands": {}
+    },
+    "nio/SNMPWalk": {
```
Comment (dorzel) [inline spec.json]: This is a duplicate entry

File: release.json (hunk 1 above)
Comment (hansmosh) [inline release.json]: Yup. Should this PR just be closed then?

Comment (dorzel) [inline release.json]: Looks like, I'll close it

**Pull request closed without merging**